src/test/java/com/tom/test/excercise01/ShoppingCartTest.java

```diff
@@ -0,0 +1,55 @@
+package com.tom.test.excercise01;
+
+import com.tom.driver.manager.DriverUtils;
+import com.tom.page.object.Excercise01Page;
+import com.tom.test.TestBase;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.math.BigDecimal;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ShoppingCartTest extends TestBase {
+
+    @Test
+    @DisplayName("Adding and removing items from cart.")
+    public void addingToCartAndRemovingFromCartItemsTest() {
+        //given
+        Excercise01Page page = new Excercise01Page();
+        int actualItemsInCart;
+        final int EXPECTED_ITEMS_IN_CART = 25;
+        BigDecimal actualPriceOfItemsInCart;
```

Comment:
move this declaration to line when you initialize them first. There is no reason to split declaration with initialization like this.